.travis.yml

```diff
@@ -1,30 +1,44 @@
-language: python
 dist: xenial
-python:
-    - "3.6"
-    - "3.7"
+sudo: required
 
-before_install:
-  - pip freeze
-  - pip uninstall numpy -y && pip install numpy==1.15.4 # FIXME: Temporary fix as f2py is buggy on numpy 1.16.0
-  - sudo apt-get install gfortran
-  - pip install Sphinx sphinx_bootstrap_theme sphinx-autodoc-typehints m2r cffi
-  - sudo apt install snapd
-  - sudo snap install dotnet-sdk --classic
-
-install:
-  - sphinx-build docs/sphinx build/html
-  - touch build/html/.nojekyll # https://help.github.com/articles/files-that-start-with-an-underscore-are-missing/
-  - cd tests/js && npm install
-  - cd $TRAVIS_BUILD_DIR
-
-script:
-  # See https://stackoverflow.com/a/34140498 for why "python -m" is needed.
-  - python3 -m pytest -v -s
-  - cd tests/js && npm test
-  - cd $TRAVIS_BUILD_DIR
-  - cd src/c_sharp && dotnet-sdk.dotnet test
-  - cd $TRAVIS_BUILD_DIR
+matrix:
+  include:
+    - language: python
+      python:
+        - "3.6"
+        - "3.7"
+      before_install:
+        - pip freeze
+        - pip uninstall numpy -y && pip install numpy==1.15.4 # FIXME: Temporary fix as f2py is buggy on numpy 1.16.0
+        - sudo apt-get install gfortran
+        - pip install Sphinx sphinx_bootstrap_theme sphinx-autodoc-typehints m2r cffi
+        - sudo apt install snapd
+        - sudo snap install dotnet-sdk --classic
+      install:
+        - sphinx-build docs/sphinx build/html
+        - touch build/html/.nojekyll # https://help.github.com/articles/files-that-start-with-an-underscore-are-missing/
+        - cd tests/js && npm install
+        - cd $TRAVIS_BUILD_DIR
+      script:
+        # See https://stackoverflow.com/a/34140498 for why "python -m" is needed.
+        - python3 -m pytest -v -s
+        - cd tests/js && npm test
+        - cd $TRAVIS_BUILD_DIR
+        - cd src/c_sharp && dotnet-sdk.dotnet test
+        - cd $TRAVIS_BUILD_DIR
+    - language: r
```

Comment:
this matrix is invalid -- at the moment this now runs only one python and one for R -- was your intention to run both `release` and `devel`. We can spawn two additional VMs for R if that is the case -- happy to make the change.

Reply:
Yes, my intention is to run both R release and R devel. Please help on how to spawn.

Reply:
Cool -- will refactor this at the end.

Reply:
Great! Thanks!

Reply:
will refactor this in a different PR later